### PR TITLE
Add prettier

### LINKED ISSUE #54: Lint (and maybe prettier)

### PR DESCRIPTION
**Pull request by andrewda**
Closes #54. This also adds a `.travis.yml` file for easy testing, along with a `yarn prettier` command for easy beautifying and a `yarn lint` command to lint the current code 👍

### CODE REVIEW

**Comment by mfix22** [comment]
Can we add `--no-semi` (:

**Comment by andrewda** [comment]
@mfix22 Done!

File: package.json [hunk 1]
```diff
@@ -7,7 +7,9 @@
     "dev": "node server.js",
     "build": "next build",
     "start": "NODE_ENV=production node server.js",
-    "deploy": "now --public"
+    "deploy": "now --public",
+    "prettier": "prettier --config .prettierrc --write 'components/**/*.js' 'handlers/**/*.js' 'lib/**/*.js' 'pages/**/*.js' 'server.js'",
```
Comment (jakedex) [inline package.json]: Any reason we can't do `*.js {components,handlers,lib,pages}/*.js` here?

Comment (andrewda) [reply package.json]: That'd be smart! Good idea!

**Comment by jakedex** [comment]
Awesome! Thanks for the help, @andrewda 😊

**Pull request merged**